presto-iceberg/src/main/java/io/prestosql/plugin/iceberg/IcebergSplitSource.java

```diff
@@ -75,7 +74,7 @@ public IcebergSplitSource(
     {
         // TODO: move this to a background thread
         List<ConnectorSplit> splits = new ArrayList<>();
-        TupleDomain<IcebergColumnHandle> predicate = convertTupleDomainTypes(this.predicate);
+        TupleDomain<IcebergColumnHandle> predicate = this.predicate;
```

Comment:
There is no need to introduce a local variable, use `predicate` member variable.

Reply:
Please remove local variable.